### PR TITLE
OWPaintData: disable zooming, have a Clear icon

### PR DESCRIPTION
**Pull request by kernc**

### CODE REVIEW

**Comment by kernc** [comment]
Coveralls really seems to not work. Both changes in this PR are in `Orange/widgets/data/`, which coverage is told to skip.

**Comment by astaric** [comment]
Clear does not seem to work while in 1d mode.

Should axis in 1d mode be shown under the points or at the bottom as it is now?

**Comment by kernc** [comment]
Fixed.

> Should axis in 1d mode be shown under the points or at the bottom as it is now?

Naaah. Having it at the bottom makes it less cluttered. :stuck_out_tongue:

**Comment by kernc** [comment]
Welcome to have another go at it.

**Comment by astaric** [comment]
1d view looks good.

Is it possible to add a bit of white space on the left and right of the plot (top and right for 2d view). The smallest and the largest value labels are only partially visible.

**Comment by astaric** [comment]
Undo of input data signal sometimes crashes the widget. To reproduce:

1. Connect iris to paint data
2. Draw something
3. Press undo (ctrl+z) twice
4. Draw something again

**Comment by kernc** [comment]
Thanks.

File: Orange/widgets/data/owpaintdata.py [hunk 1]
```diff
@@ -27,6 +24,9 @@
 from Orange.widgets.utils import itemmodels, colorpalette
 from Orange.widgets.io import FileFormat
 
+from Orange.util import scale, namegen
+import pyqtgraph as pg
```
Comment (astaric) [inline Orange/widgets/data/owpaintdata.py]: Imports of external libraries are generally above imports of Orange.

Comment (kernc) [inline Orange/widgets/data/owpaintdata.py]: The module, as was, failed to run as "__main__" with latest pyqtgraph release. :(

Comment (kernc) [inline Orange/widgets/data/owpaintdata.py]: Is reverted.

**Pull request merged**